pyext/src/analysis_trajectories.py

```diff
@@ -395,7 +443,7 @@ def read_stats_detailed(self,
             info_names, info_fields = self.get_info_fields(stat2_dict)
 
             line_number=0
-            for line in open(file).readlines():
+            for line in open(sf).readlines():
```

Comment:
Perhaps out of scope of this patch, but in general avoid `open(foo).readlines()`. Write as

    with open(foo) as fh:
        for line in fh.readlines():

so that `fh` is definitely closed at the end of the loop (may not be otherwise in Python implementations that don't  use refcounting for garbage collection).

Reply:
all file opening everywhere in ```analysis_trajectories``` now follow this pattern.